app/mod_blog/forms.py

```diff
@@ -2,7 +2,7 @@
 
 from flask_wtf import Form
 from wtforms_components import DateRange
-from wtforms import StringField,BooleanField,PasswordField,SubmitField,TextAreaField,SelectField,DateField
+from wtforms import StringField,BooleanField,PasswordField,SubmitField,TextAreaField,SelectField,DateField,HiddenField
```

Comment:
Write the import statement in multi-lined if like :  from wtforms import (
       StringField, BooleanField, PasswordField, SubmitField, TextAreaField, SelectField,  DateField, HiddenField
)